src/widget/translator.cpp

```diff
@@ -72,6 +73,11 @@ void Translator::translate()
         QCoreApplication::installTranslator(translator);
     }
 
+    // Added to fix #3542
+    QSet<QString> rtlLanguages = {"ar", "he"};
```

Comment:
hardcoding languages doesn't seem like a good idea, especially given that every time a rtl language will be added, this will need to be updated